### DISCUSSION TITLE
instalar pugin en Ubuntu

### DISCUSSION THREAD

**Discussion by ZukoHama** [sorayuki/obs-multi-rtmp#79] [Q&A]
Como puedo instalar el plugin para trasmitir en varios sitios, estoy habalndo de obs-multi-rtmp pero lo que no se es como instalarlo en ubuntu gracias por que todos los que aparecen para descargar son para windows....el que me pueda ayudar

**Comment by AntonioGomes42** [comment]
This is a tutorial that i made, showing how to install on ubuntu, i hope it can help you: https://www.youtube.com/watch?v=A1HZEBA3Rl8

**Reply by renzo01** [reply]
I saw that video, but when I execute build_linux, it show me an Error

CMake Error at CMakeLists.txt:13 (find_package):
  By not providing "FindQt6.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Qt6", but
  CMake did not find one.

  Could not find a package configuration file provided by "Qt6" with any of
  the following names:

    Qt6Config.cmake
    qt6-config.cmake

  Add the installation prefix of "Qt6" to CMAKE_PREFIX_PATH or set "Qt6_DIR"
  to a directory containing one of the above files.  If "Qt6" provides a
  separate development package or SDK, be sure it has been installed.


-- Configuring incomplete, errors occurred!
See also "/home/banak/obs-multi-rtmp/build/CMakeFiles/CMakeOutput.log".

**Comment by AntonioGomes42** [comment]
I made a new tutorial showing how to install it on a Flatpak OBS. I works for every single linux. Link : https://youtu.be/p4eoEoq438M. I hope it helps you!
